components/kyma-environment-broker/internal/avs/client.go

```diff
@@ -96,6 +96,39 @@ func (c *Client) AddTag(evaluationID int64, tag *Tag) (*BasicEvaluationCreateRes
 	return &responseObject, nil
 }
 
+func (c *Client) SetStatus(evaluationID int64, status string) (*BasicEvaluationCreateResponse, error) {
+	var responseObject BasicEvaluationCreateResponse
+
+	objAsBytes, err := json.Marshal(status)
+	if err != nil {
+		return &responseObject, errors.Wrap(err, "while marshaling SetStatus request")
+	}
+	absoluteURL := appendId(c.avsConfig.ApiEndpoint, evaluationID)
+
+	request, err := http.NewRequest(http.MethodPost, fmt.Sprintf("%s/lifecycle", absoluteURL), bytes.NewReader(objAsBytes))
+	if err != nil {
+		return &responseObject, errors.Wrap(err, "while creating SetStatus request")
+	}
+	request.Header.Set("Content-Type", "application/json")
+
+	response, err := c.execute(request, false, true)
+	if err != nil {
+		return &responseObject, errors.Wrap(err, "while executing SetStatus request")
+	}
+	defer func() {
+		if closeErr := c.closeResponseBody(response); closeErr != nil {
```

Comment:
Misc bug: `closeResponseBody()` doesn't drain the response body before closing it.